src/microbe_stage/patch.h

```diff
@@ -83,11 +83,22 @@ class Patch : public Leviathan::ReferenceCounted {
         return adjacentPatches;
     }
 
+    //get and set coordinates for the patch to be displayed in the gui
+    bool
+        setScreenCoordinates(Float2 coordinates);
+
+    Float2
+        getScreenCoordinates() const
+    {
+        return screenCoordinates;
+    }
+
     REFERENCE_COUNTED_PTR_TYPE(Patch);
 
 private:
     const int32_t patchId;
     std::string name;
+    Float2 screenCoordinates; //where the patch should be displayed in the gui.
```

Comment:
This comment should be like this to work with doxygen:
```
//! Where the patch should be displayed in the gui.
Float2 screenCoordinates;

Reply:
Yeah ok I can change this tomorrow,